### PR TITLE
Group 26 user should be able to post a picture to predict the disease of the crop

### PR DESCRIPTION
**Pull request by harish426**
Frontend part for prediction feature completed. Check any changes required and some changes have been made for feed for good looking using css

### CODE REVIEW

File: Frontend/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -3,12 +3,18 @@ import { RouterModule, Routes } from '@angular/router';
...
 import { LoginComponent } from './login/login.component';
 import { FeedComponent } from './feed/feed.component';
+import { DetectComponent } from './detect/detect.component'
 import { AuthGuard } from './helpers/auth.guard';
 
 const routes: Routes = [
   { path: '', component: LoginComponent },
+  { path: 'home', component: HomeComponent },
+  { path: 'feed', component: FeedComponent },
+  { path: 'detect', component: DetectComponent},
```
Comment (udkreddy) [inline Frontend/src/app/app-routing.module.ts]: Add authgaurd as similar to the 15th line

Comment (udkreddy) [reply Frontend/src/app/app-routing.module.ts]: **  canActivate: [AuthGuard]

File: Frontend/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -3,12 +3,18 @@ import { RouterModule, Routes } from '@angular/router';
 import { HomeComponent } from './home/home.component';
 import { LoginComponent } from './login/login.component';
 import { FeedComponent } from './feed/feed.component';
+import { DetectComponent } from './detect/detect.component'
 import { AuthGuard } from './helpers/auth.guard';
 
 const routes: Routes = [
   { path: '', component: LoginComponent },
+  { path: 'home', component: HomeComponent },
+  { path: 'feed', component: FeedComponent },
```
Comment (udkreddy) [inline Frontend/src/app/app-routing.module.ts]: This can be removed

**Comment by udkreddy** [comment]
Added review comments, please check @harish426

File: Frontend/src/app/app-routing.module.ts (hunk 2 above)
Comment (harish426) [reply Frontend/src/app/app-routing.module.ts]: removed

File: Frontend/src/app/app-routing.module.ts (hunk 1 above)
Comment (harish426) [reply Frontend/src/app/app-routing.module.ts]: fixed

**Review by udkreddy — APPROVED** [review]

**Pull request merged**